recipes/alteryx-open-src-update-checker/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "alteryx-open-src-update-checker" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/alteryx-open-src-update-checker/alteryx_open_src_update_checker-{{ version }}.tar.gz
+  sha256: 814003ab6952c406263643ea6dc9d37049ba2d53bab91775d383ebb08e39c894
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  script_env:
+    - ALTERYX_OPEN_SRC_UPDATE_CHECKER=False
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - requests >=2.22.0
+test:
+  imports:
+    - alteryx_open_src_update_checker
+  requires:
+    - pytest ==5.0.0
```

Comment:
The pytest version is actually different because the one in the gh repo would cause a version conflict